.github/COPILOT_CUSTOMIZATION.md

```diff
@@ -0,0 +1,146 @@
+  - Focus: Lit components, file structure, naming conventions, Atomic Chemistry
+
+- **tests-atomic.instructions.md** - Testing conventions for Atomic package
+  - Applies to: `**/atomic/**/*.spec.{ts}`
+  - Focus: Vitest test patterns, test structure, best practices
+
+### Technology-Specific Instructions
+
+- **a11y.instructions.md** - Accessibility guidelines (WCAG 2.2 Level AA)
+  - Applies to: `**` (all files)
+  - Focus: Accessible code generation, keyboard navigation, screen reader support
+
+- **playwright-typescript.instructions.md** - Playwright E2E test writing
+  - Applies to: `**` (all files)
+  - Focus: Test structure, locators, assertions, best practices
+
+- **typescript-5-es2022.instructions.md** - TypeScript 5.x development guidelines
+  - Applies to: `**/*.ts`
+  - Focus: Type safety, ES2022 features, naming conventions, architecture
+
+## Chatmodes
+
+Chatmodes provide specialized AI personas for specific tasks. To use a chatmode in VS Code:
+
+1. Open GitHub Copilot Chat
+2. Reference the chatmode using `@chatmode` or by selecting it from the UI
+
+Available chatmodes:
+
+### Development Chatmodes
+
+- **accessibility.chatmode.md** - Accessibility-focused development mode
+  - Focus: WCAG 2.1 compliance, testing with pa11y and axe-core
+  - Tools: Code editing, testing, browser preview
+
+- **typescript-mcp-expert.chatmode.md** - TypeScript MCP server expert
+  - Focus: Model Context Protocol server development
+  - Best for: Building MCP servers, TypeScript patterns
+
+### Project Management Chatmodes
+
+- **refine-issue.chatmode.md** - Issue refinement and enrichment
+  - Focus: Adding acceptance criteria, technical considerations, edge cases
+  - Tools: GitHub issue management
+
+- **research-technical-spike.chatmode.md** - Technical spike research
+  - Focus: Systematic investigation, documentation research, experimental validation
+  - Tools: Search, code analysis, documentation
+
+- **task-researcher.chatmode.md** - Deep task analysis and research
+  - Focus: Comprehensive project analysis, alternative evaluation
+  - Tools: All available tools for research
+
+## Prompts
+
+The prompts directory contains reusable prompts for common tasks:
+
+- **generate-vitest-tests-atomic-lit-components.prompt.md** - Generate Vitest tests for Lit components
+- **migrate-stencil-to-lit.prompt.md** - Migrate Stencil components to Lit
+- **generate-vitest-tests-atomic-utils.prompt.md** - Generate tests for utility files
+
+## How to Use
+
+### Using Instructions
+
+Instructions are automatically applied by GitHub Copilot based on the file you're editing. No action required!
+
+### Using Chatmodes
+
+In VS Code with GitHub Copilot:
+
+1. Open Copilot Chat (Ctrl/Cmd + Shift + I)
+2. Type `/` to see available commands
+3. Select or reference a chatmode for specialized assistance
+
+### Using Prompts
+
+Prompts can be referenced in Copilot Chat or used as templates for common tasks.
+
+## Repository-Specific Guidelines
```

Comment:
I'm a bit confused: this document seems like it's overall intended for human readers (not AI), but this section is kind of out of scope. It's the kind of info I'd expect to see in the main README for the project (although it has information that's specific Atomic too...).

The information isn't wrong, it's just awkward in this specific file.